homeassistant/components/rainmachine/const.py

```diff
@@ -17,3 +18,10 @@
 
 DEFAULT_PORT = 8080
 DEFAULT_ZONE_RUN = 60 * 10
+
+
+@dataclass
+class RainMachineSensorDescriptionMixin:
+    """Define an entity description mixin for binary and regular sensors."""
+
+    api_category: str
```

Comment:
Some integrations put these in a `models.py` file. Might be something worth considering.